rero_mef/agents/mef/api.py

```diff
@@ -166,6 +168,20 @@ def create_or_update_mef_viaf_record(self, dbcommit=False, reindex=False,
         """
         return self, Action.ERROR, None, False
 
+    @classmethod
+    def create_deleted(cls, record, dbcommit=False, reindex=False):
+        """Create a deleted record for an record.
+
+        :param record: Record to create.
+        :param dbcommit: Commit changes to DB.
+        :param reindex: Reindex record.
+        :returns: Created record.
+        """
+        data = {record.name: {
+            '$ref': build_ref_string(record.pid, record.name)}}
+        data['deleted'] = datetime.now(timezone.utc).isoformat()
```

Comment:
could be merge to increase readability (my opinion)
```python
data = {
    record.name: {
        '$ref': build_ref_string(record.pid, record.name)
    },
    deleted: datetime.now(timezone.utc).isoformat()
}
```
This method could be placed into the parent `EntityMefRecord` to declare it once and use them in all children classes.